lib/foreman_puppet/engine.rb

```diff
@@ -60,6 +60,7 @@ class Engine < ::Rails::Engine
       ::Api::V2::HostgroupsController.include ForemanPuppet::Extensions::ParametersHostgroup
       ::OperatingsystemsController.prepend ForemanPuppet::Extensions::OperatingsystemsController
       ::HostsController.include ForemanPuppet::Extensions::HostsControllerExtensions
+      ::ComputeResourcesVmsController.include ForemanPuppet::Extensions::ComputeResourcesVmsController
```

Comment:
Wouldn't simple
```suggestion
      ::ComputeResourcesVmsController.helper ForemanPuppet::HostsAndHostgroupsHelper
```
work here as well? Then we would do without an extension for the controller.

Reply:
Thanks @ezr-ondrej it works, and code looks better 👍🏼